mediawiki/LocalSettings.php

```diff
@@ -36,26 +36,17 @@
 
 ## The URL paths to the logo.  Make sure you change this from the default,
 ## or else you'll overwrite your logo when you upgrade!
-#$wgLogos = [
-#	'1x' => "/themes/images/logo_wikifavelas2024.svg",
-#	'wordmark' => [
-#		"src" => "/themes/images/logo_wikifavelas2024.svg",
-#		"width" => 119,
-#		"height" => 18,
-#	],
-#	'icon' => "/themes/images/logo_wikifavelas.svg",
-#];
-$wgLogos = [ '1x' => "$wgResourceBasePath/themes/images/logo_wikifavelas2024.svg" ];
-
-
+$wgLogos = [
+	'1x' => "$wgResourceBasePath/themes/images/logo_white.png",
+	'icon' => "$wgResourceBasePath/resources/assets/change-your-logo-icon.svg",
```

Comment:
Aqui acho que pode ser o nome hardcode mesmo, algo como:


```suggestion
	'icon' => "$wgResourceBasePath/resources/assets/logo.svg",
```

Assim fica algo genérico para os dois projetos.